src/Command/ErrorFormatter/GithubErrorFormatter.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command\ErrorFormatter;
+
+use PHPStan\Command\AnalysisResult;
+use PHPStan\Command\Output;
+use PHPStan\File\RelativePathHelper;
+
+class GithubErrorFormatter extends TableErrorFormatter
```

Comment:
Composition over inheritance should be used. Ask for TableErrorFormatter in the constructor, do not extend from it.